app/modules/ph/models_ph.py

```diff
@@ -0,0 +1,14 @@
+from datetime import date
+
+from sqlalchemy import Date, String
+from sqlalchemy.orm import Mapped, mapped_column
+
+from app.types.sqlalchemy import Base
+
+
+class Paper(Base):
+    __tablename__ = "papers"
```

Comment:
Please prefix the table name with `ph` to prevent collision with other modules
```suggestion
    __tablename__ = "ph_paper"
```